test/gcs.js

```diff
@@ -165,37 +165,36 @@ describe('UploadFS GCS', function () {
     uploadfs.copyImageIn('test.jpg', imgDstPath, (e, info) => {
       assert(!e, 'gcs copyImageIn works');
 
-      setTimeout(() => {
-        const url = uploadfs.getUrl();
-        const paths = [ info.basePath + '.jpg' ];
+      const url = uploadfs.getUrl();
+      const paths = [ info.basePath + '.jpg' ];
 
-        paths.push(info.basePath + '.small.jpg');
-        paths.push(info.basePath + '.medium.jpg');
-        paths.push(info.basePath + '.large.jpg');
+      paths.push(info.basePath + '.small.jpg');
+      paths.push(info.basePath + '.medium.jpg');
+      paths.push(info.basePath + '.large.jpg');
 
-        async.map(paths, (path, cb) => {
-          const imgPath = url + path;
+      async.map(paths, (path, cb) => {
+        const imgPath = url + path;
 
-          fetch(imgPath, {
-            method: 'GET'
+        fetch(imgPath, {
+          method: 'GET'
+        })
+          .then(function(res) {
+            assert(res.status === 200, `Request status 200 != ${res.status}`);
+            return res.text();
```

Comment:
This was the important change. If we don't make the call to fetch the actual content, the `fetch` request never terminates, so the test suite never exits.